pythonsdk/src/main/python/vantiqservicesdk.py

```diff
@@ -149,13 +150,19 @@ def __complete_request(completed_task):
                         continue
 
                     # Spawn a task to process the message and send the response
-                    task = asyncio.create_task(self.__process_message(websocket, msg_bytes))
-
-                    # Add the task to the set of active requests and remove when done.
-                    # See https://docs.python.org/3/library/asyncio-task.html#creating-tasks
-                    active_requests.add(task)
-                    self._active_requests.inc(1)
-                    task.add_done_callback(__complete_request)
+                    request = json.loads(msg_bytes.decode("utf-8"))
```

Comment:
could not see a way to determine options without decoding here (instead of in __process_message)